src/main/java/com/easyrest/components/BaseComponent.java

```diff
@@ -0,0 +1,18 @@
+package com.easyrest.components;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class BaseComponent {
+    protected WebDriver driver;
+
+    public BaseComponent(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public void enterData(String data, By element, WebDriver driver){
```

Comment:
It is not correct to have a public method with By and WebDriver parameters because it breaks the encapsulation principle. Please specify another access modifier for that method.